packages/contracts/contracts/THUSDOwner.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+
+import "./Interfaces/ITHUSDToken.sol";
+import "./Dependencies/CheckContract.sol";
+import "./Dependencies/Ownable.sol";
+
+contract THUSDOwner is Ownable, CheckContract {
+    ITHUSDToken private thusdToken;
+
+    address public governorBravoAddress;
```

Comment:
```suggestion
    address public immutable governorBravoAddress;
```

Reply:
Addressed in [refactor: remove param of transfer thusd ownership function](https://github.com/Threshold-USD/dev/pull/108/commits/3371b2b661ab1d729d168e59114e7555f7221421)